pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -239,13 +248,9 @@ type ScorePlugin interface {
 	// the pod will be rejected.
 	Score(pc *PluginContext, p *v1.Pod, nodeName string) (int, *Status)
 
-	// NormalizeScore is called for all node scores produced by the same plugin's "Score"
-	// method. A successful run of NormalizeScore will update the scores list and return
-	// a success status.
-	//
-	// NOTE: This function is optional, and you could implement it as a no-op by simply
-	// returning nil.
-	NormalizeScore(pc *PluginContext, p *v1.Pod, scores NodeScoreList) *Status
+	// Extensions returns an extension if the plugin implements one, or nil if it
+	// does not.
```

Comment:
I would phrase it: "returns a ScoreExtensions interface if it implements one, or nil if does not"